brightchain-lib/src/lib/documents/document.ts

```
@@ -0,0 +1,300 @@
+    const blockSize = BlockService.getBlockSizeForData(json.length);
+    const xorResult = SerializableBuffer.alloc(blockSize as number);
+    let firstBlock = true;
+    await BlockService.processFileInChunks(
+      json,
+      blockSize,
+      TUPLE.SIZE,
+      async (chunkDatas: SerializableBuffer[]) => {
+        if (firstBlock) {
+          chunkDatas[0].copy(xorResult, 0, 0, blockSize as number);
+        }
+        for (let i = (firstBlock ? 1 : 0); i < chunkDatas.length; i++) {
```

ESLint

[error] Replace `(firstBlock·?·1·:·0)` with `firstBlock·?·1·:·0`

(prettier/prettier)